lib/util/html.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+//
+// Helper functions that relate to the HTTP/service layer of the application.
+
+// handy dev function for enabling syntax hilighting of html
+const html = ([ first, ...rest ], ...vars) => first + vars.map((v, idx) => [ v, rest[idx] ]).flat().join('');
+
+// Style to look like odk-central-frontend
```

Comment:
@matthew-white can you share some input here?  How should this page look?

Reply:
I had an idea about this [above](https://github.com/getodk/central-backend/pull/910#discussion_r1277685758):

> For the success page, what do you think about leaving that as unstyled as possible and not trying to make it look like Frontend? Since the user will only be on the page for a short time and is automatically redirected, I don't feel like it needs to be styled. I've seen that pattern elsewhere and feel like it wouldn't be overly confusing to users.

I think trying to match the style of Frontend would lead to a maintenance burden. Frontend styles will inevitably change over time, and it'd be good not to have to change them in two places.

Reply:
> Frontend styles will inevitably change over time

How frequently?

I think a branded forwarding page maintains users' trust that they are inside a legitimate workflow.

Reply:
> How frequently?

Styles that would affect this page: likely not often, but definitely sometimes.

Already, the styles don't quite match. There's a max width on `.panel-main` that seems to be missing here. You can see that in a wider viewport. Here's a panel as rendered by Frontend, where you can see the max width:

<img width="1900" src="https://github.com/getodk/central-backend/assets/5970131/40dd2e7f-79af-444a-9291-36d82f6e4a82">

Here the max width is missing:

<img width="1710" src="https://github.com/getodk/central-backend/assets/5970131/c00a2c4a-214c-4911-8f1a-94ccfd769cea">

That change happened in getodk/central-frontend#775. Follow-up work is planned at getodk/central-frontend#785, which is an example of how styles evolve over time.

> I think a branded forwarding page maintains users' trust that they are inside a legitimate workflow.

If it didn't come with any cost, I'd be in favor of it. But given that it requires ongoing maintenance that will be easy to forget, I don't think we should render a page like that. Redirect pages are often styled very simply, so I don't expect that user trust will be broken.

This definitely shouldn't block merging though. I'd be happy to file an issue and continue discussing there.

Reply:
I've created an issue for this: #971